notebooks/3-Model-pipeline.ipynb

```diff
@@ -49,7 +49,7 @@
     }
```

Comment:
<p>The answer to the exercise isn't really correct anymore. The highest values aren't necessarily on the diagonal.</p><p> </p><p>I propose a new answer:</p><p>While the majority of the false predictions are done correctly (true negatives), it is relatively bad at predicting positive labels. This could be due to the fact that the dataset is skewed.</p><p> </p><p>However, we predict on the same data that we trained on. To get truly a fair estimate of the perfomance, we need to apply it on a new data set, such as the test set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>